### PR TITLE
shell game added with updated readme

### PR DESCRIPTION
**Pull request by Tamilkumaran17**
### 👨‍💻 Changes proposed and Brief Description

I have added the shell game and also updated the README content and also updated the index.html with my file link.

### ✅ Check List (Check all the applicable boxes) 

- [ ✅] My code doesn't break any part of the project
- [ ✅] This PR does not contain plagiarized content.
- [✅ ] My Addition/Changes works properly and matches the overall repo pattern.
- [✅ ] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
Screenshot (136)

### CODE REVIEW

**Comment by Tamilkumaran17** [comment]
Can you please explain if any conflict may occurs, i will be very useful to me to resolve it!

File: index.html
Comment (swapnilsparsh) [inline index.html:1]: Don't remove any project, just add your project.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]

**Comment by Tamilkumaran17** [comment]
i have resolved the changes. can you check if any conflicts occurs?

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
When someone fails to find the ball for the first time then game should end instead it does not end right not.

**Comment by Tamilkumaran17** [comment]
now the game conditions are changed according to your constrains. can you check for any conflicts?

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
1. You have just fixed the text and nothing else.
2. Everything is static no photos or anything else to make it look good
3. When game is over then user should not be able to choose other options and "play again" button should be there

**Comment by Tamilkumaran17** [comment]
here i have added the play again button as you mentioned with small animations.

File: 30DaysOfJavaScript/assets/116.png
Comment (swapnilsparsh) [inline 30DaysOfJavaScript/assets/116.png:1]: Everything is working fine just update the screenshot

Comment (Tamilkumaran17) [reply 30DaysOfJavaScript/assets/116.png:1]: updated screenshot in assets file.

Comment (swapnilsparsh) [inline 30DaysOfJavaScript/assets/116.png:1]: Image should not contain the browser and taskbar. Only project screenshot should be there.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]

File: 30DaysOfJavaScript/assets/116.png
Comment (Tamilkumaran17) [reply 30DaysOfJavaScript/assets/116.png:1]: updated!!

**Review by swapnilsparsh — APPROVED** [review]

**Comment by swapnilsparsh** [comment]
@Tamilkumaran17 Thank you for contributing to this repo. Looking forward for more 🚀

**Pull request merged**